protocol/x/revshare/keeper/revshare.go

```diff
@@ -163,25 +163,25 @@ func (k Keeper) GetAllRevShares(
 	feeSourceToRevSharePpm := make(map[types.RevShareFeeSource]uint32)
 	feeSourceToQuoteQuantums[types.REV_SHARE_FEE_SOURCE_TAKER_FEE] = big.NewInt(0)
 	feeSourceToRevSharePpm[types.REV_SHARE_FEE_SOURCE_TAKER_FEE] = 0
-	feeSourceToQuoteQuantums[types.REV_SHARE_FEE_SOURCE_NET_FEE] = big.NewInt(0)
-	feeSourceToRevSharePpm[types.REV_SHARE_FEE_SOURCE_NET_FEE] = 0
+	feeSourceToQuoteQuantums[types.REV_SHARE_FEE_SOURCE_NET_PROTOCOL_REVENUE] = big.NewInt(0)
+	feeSourceToRevSharePpm[types.REV_SHARE_FEE_SOURCE_NET_PROTOCOL_REVENUE] = 0
 
 	totalFeesShared := big.NewInt(0)
 	takerFees := fill.TakerFeeQuoteQuantums
 	makerFees := fill.MakerFeeQuoteQuantums
 	netFees := big.NewInt(0).Add(takerFees, makerFees)
 
-	affiliateRevShares, err := k.getAffiliateRevShares(ctx, fill, affiliatesWhitelistMap)
+	affiliateRevShares, affiliateFeesShared, err := k.getAffiliateRevShares(ctx, fill, affiliatesWhitelistMap)
 	if err != nil {
 		return types.RevSharesForFill{}, err
```

Comment:
Comment on caller function `persistMatchedOrders`:

Instead propagating error returned from `GetAllRevShares` (which would ultimately fail the match transaction), it's better to just pass in empty `RevSharesForFill` to `DistributeFees`. This way if any rev share calculation encounters issue, the chain will continue to be able to process matches (without rev share). 

Order matching + rev share both working > Only order matching working > Order matching not working

Reply:
sounds good. I'll add some error logs to catch in case it fails

Reply:
Did we do this in this PR? Or plan to do in another one?

Reply:
This is done in the PR . Check [here](https://github.com/dydxprotocol/v4-chain/blob/e69bbb77a7e9edcfbd4ec2bf6a848d483187c30f/protocol/x/revshare/keeper/revshare.go#L181-L195). Also updated tests in revshare_test.go

Reply:
Oh, the original comment was suggesting that we keep `GetAllRevShares` behavior as this but handle error differently in `persistMatchedOrders`, since it's the caller responsibility to determine what to do with the returned error from `GetAllRevShares` 

Changing `GetAllRevShares` behavior works too but we are changing the signature of the function. Also it doesn't make sense to have this function return error anymore. 

We should either:
- Keep `GetAllRevShares` as is, and just log out error (don't propogate errors further up) in `persistMatchedOrders`.
- Fully change signature of `GetAllRevShares` so it doesn't return errors anymore, maybe even rename it to `GetAllValidRevShares`

The first seems a bit cleaner to me since we just need one error log instead of multiple